src/fidesctl/core/dataset.py

```diff
@@ -86,8 +86,9 @@ def create_db_dataset(schema_name: str, db_tables: Dict[str, List[str]]) -> Data
 
     Placeholder values are use where needed.
     """
+    validated_fides_key = check_fides_key(schema_name)
```

Comment:
I see that you used `sanitize_fides_key` as another function, but when I see `validated_fides_key` I think that it checks to make sure that it is compatible and throws an error if it isn't, I wouldn't expect this to mean that it has actually sanitized the key (although that is what is happening), so I think we should be a little more clear with our naming here

Reply:
makes sense, good call 👍🏽 I was thinking about the future check for duplicate naming here as well and sort of trying to capture a wider view but it's not necessary at this point so will sanitize this name 😁